### ISSUE TITLE
Fallbacks for context-vars in cccid?

### ISSUE THREAD

**Issue by dbr** [imageworks/OpenColorIO#207]
Currently I have a directory containing a few .ccc files, like `/.../grades/neutral.ccc` `/.../grades/beauty.ccc`

I refer to these in the config as

```
- !<Look>
  name: beauty
  process_space: jplog
  transform: !<FileTransform> {src: "beauty.ccc", cccid: "${SEQ}/${SHOT}"}
```

This works fine, but errors if the cccid doesn't exist

Upon typing this I realise that storing the grades as `.cc` and having multiple paths on the search_path is a much better idea (as described on #201), since it allows for things such as a default sequence grade, and is possibly faster since it doesn't have to load a large XML file just to get one grade

..but I'm curious if there's any way to achieve something similar with cccid's?

**Comment by scoopxyz** [comment]
Is this successfully addressed in the closed #346 ?

**Comment by scoopxyz** [comment]
@dbr pinging this again

**Comment by dbr** [comment]
Not sure it was that specific pull request, but this problem seems to have been fixed in e48cce1653c45525652a3c69ab469d181e40467a (which is part of v1.0.9) \o/
